specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json

```diff
@@ -156,6 +156,70 @@
           }
         ]
       }
+    },
+    "/providers/Microsoft.Subscription/subscriptions/{subscriptionId}/cancel": {
```

Comment:
To create a subscription, the createSubscription action permission is needed at the scope of /providers/Microsoft.Billing/billingAccounts/{billingAccountName}/invoiceSections/{invoiceSectionName}.
Why isnt the cancel permission needed at the same scope? If the permission needs to be the same, then should the URL be similar to createSubscription

Reply:
Operations under /providers/Microsoft.Billing are meant for people with billing access to things
whereas operations under /providers/Microsoft.Subscription are meant for service administrators. Here we allowing service administrators to cancel/rename subscriptions

Reply:
Yes, I understand, but Microsoft.Subscription RP (service) exposes the operation to create a subscription which is an extension of /providers/Microsoft.Billing/billingAccounts/{billingAccountName}/invoiceSections/{invoiceSectionName}
Why isnt same permission needed to cancel or rename subscription?

Reply:
It is the approach that we are following in GTM, that we want to allow only the service people to access the APIs that have service impact. We don't want billing people to cancel the API who are not aware of the usage aspects of the subscription. But billing people would be allowed to stop the billing if they want.

Reply:
Ok.